test/unit/services/JobRoleServiceTests.ts

```diff
@@ -0,0 +1,55 @@
+import axios from "axios";
+import MockAdapter from "axios-mock-adapter";
+import { expect } from 'chai';
+import { JobRoleResponse } from "../../../src/models/JobRoleResponse";
+import { getAllJobRoles, URL } from "../../../src/services/JobRoleService";
+
+
+const jobRoleResponse: JobRoleResponse = {
+    jobRoleId: 1,
+    roleName: "delivery_master",
+    jobRoleLocation: "Buenos Aires",
+    capabilityName: "delivery",
+    bandName: "Architect",
+    closingDate: new Date("10/10/2024")
+}
+
+const mock = new MockAdapter(axios);
+
+describe('JobRoleService', function () {
+
+    describe('getAllJobRoles', function () {
+      it('should return job roles from response', async () => {
+        const data = [jobRoleResponse];
+
+        mock.onGet(URL).reply(200, data);
+
+        const results = await getAllJobRoles();
+        results[0].closingDate = new Date(results[0].closingDate);
+
+        expect(results[0]).to.deep.equal(jobRoleResponse);
+      })
+
+      it('should throw exception when 500 error returned from axios', async () => {
+        mock.onGet(URL).reply(500);
+
+        try {
+          await getAllJobRoles();
```

Comment:
please add expect true to be false or false to be true after each expression that should return the error -> if error is not thrown, test will fail. Right now false pass is possible